### PR TITLE
Improvement/zenko 4737 add e2e test for dmf retries

### PR DESCRIPTION
**Pull request by KillianG**
ZENKO-4737

### CODE REVIEW

File: tests/ctst/steps/dmf.ts [hunk 1]
```diff
@@ -16,3 +17,12 @@ Then('dmf volume should contain {int} objects', async (objectCount: number) => {
...
     assert(conditionOk);
 });
+
+Given('a flaky backend that will require {int} retries for {string}',
+    function (this: Zenko, retryNumber: number, op: string) {
+        assert(["restore", "archive", "command"].includes(op), `Invalid operation ${op}`);
+        assert(retryNumber > 0, `Invalid retry number ${retryNumber}`);
+
+        this.addToSaved('backendFlakynessRetryNumber', retryNumber);
+        this.addToSaved('backendFlakyness', op);
```
Comment (welansari) [inline tests/ctst/steps/dmf.ts]:
```suggestion
        this.addToSaved('backendFlakinessRetryNumber', retryNumber);
        this.addToSaved('backendFlakiness', op);
```
nit

**Review by welansari — APPROVED** [review]

File: tests/ctst/steps/dmf.ts [hunk 2]
```diff
@@ -16,3 +17,12 @@ Then('dmf volume should contain {int} objects', async (objectCount: number) => {
     }
     assert(conditionOk);
 });
+
+Given('a flaky backend that will require {int} retries for {string}',
+    function (this: Zenko, retryNumber: number, op: string) {
+        assert(["restore", "archive", "command"].includes(op), `Invalid operation ${op}`);
```
Comment (ghost) [inline tests/ctst/steps/dmf.ts]:
```suggestion
        assert(['restore', 'archive', 'command'].includes(op), `Invalid operation ${op}`);
```

**Review by ghost — APPROVED** [review]

**Pull request merged**